src/main/java/Command.java

```diff
@@ -0,0 +1,5 @@
+public abstract class Command {
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException;
+
+    public abstract boolean isExit();
```

Comment:
You might want to consider setting `isExit` to `False` in the abstract, then only changing it to `True` when it is an ExitCommand. This way you do not have to constantly check what Command it is